env.default

```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
 # See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
 BASKET_API_ROOT_URL=https://basket-dev.allizom.org/
 
+# CSP config
+CSP_CONNECT_SRC=" * "
+CSP_FONT_SRC=" 'self' fonts.googleapis.com fonts.gstatic.com"
+CSP_FORM_ACTION=" 'self' https://webto.salesforce.com/ mozillasalescloud--mkl.cs4.my.salesforce.com "
+CSP_FRAME_SRC=" 'self' assets.braintreegateway.com c.paypal.com *.paypal.com https://www.google.com/recaptcha/api2/ https://tst.kaptcha.com/ "
+CSP_IMG_SRC=" * data: "
```

Comment:
Do we use data: urls for images on donate? If so, then this makes sense, but do we?